gen-ai/orchestrator-server/src/main/python/server/pyproject.toml

```diff
@@ -20,6 +20,8 @@ colorlog = "^6.8.0"
 boto3 = "^1.34.25"
 urllib3 = "^1.26"
 jinja2 = "^3.1.3"
+flashrank = "^0.2.6"
```

Comment:
For langchain dependency could you for now point to your fork and your branch used in this PR : https://github.com/langchain-ai/langchain/pull/24016/files 

See :  https://stackoverflow.com/a/69124405